exercises/concept/secrets/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+# Instructions
+
+In this exercise, you've been tasked with writing the software for an encryption device that works by performing transformations on data. You need a way to flexibly create complicated functions by combining simpler functions together.
+
+For each task, return a function (using a `lambda`).
+
+
+## 1. Create an adder
+
+Implement `secret_add`. It should return a function which takes one argument and adds to it the argument passed in to `secret_add`.
+
+```python
+>>> adder = secret_add(2)
+>>> adder(2)
+4
+
+## 2. Create a multiplier
+
+Implement `secret_multiply`. It should return a function which takes one argument and multiplies it by the secret passed in to `secret_multiply`.
+
+```python
+multiplier = secret_multiply(7)
+multiplier(3)
+# => 21
+```

```

Comment:
The remainder of these code examples should also be changed per the code block above.

Reply:
You're late. I already did. :)